core/model.py

```diff
@@ -46,6 +45,11 @@ def __init__(self, path=DEFAULT_MODEL_PATH):
     def _predict(self, model):
         logger.info('Generating audio from model: {}'.format(model))
         preds = self.models[model].predict()
-        write_wav('output.wav', preds[0], 16000)
+
+        # convert audio to 16 bit so that it can play in firefox
+        audio_data = np.round(preds[0] * 32767)
+        audio_data = audio_data.astype(np.int16)
```

Comment:
This may be more efficient:

```python
audio_data = np.empty_like(dtype=np.int16)
np.round(preds[0] * np.iinfo(np.int16).max, out=audio_data)
```

Reply:
Resolving this for now based on offline discussion